_roadmap/01_plans.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: single
+sidebar:
+  nav: plans.md
+title: Plans for upcoming challenges
+excerpt: Plans for upcoming SAMPL challenges and phases
+permalink: /roadmap/plans/
+---
+
+## SAMPL overview, SAMPL7-10
+
+As of SAMPL7-10, our NIH funding allows us to be somewhat more strategic in planning challenges, so we plan a series of challenges encompassing:
+- Physical property prediction (e.g. pKa, logP, logD, etc.)
+- Host-guest binding
+- Protein-ligand binding
+
+SAMPL is now broken into phases, each running roughly a year, with each phase encompassing several component challenges. [Workshop plans](https://samplchallenges.github.io/current/workshops/) are detailed separately.
+
+The Gibb and Isaacs groups, focusing on host-guest binding, plan to collect regular data on host-guest binding, and protein-ligand binding assays are coming online in the Chodera lab. Physical property data, however, relies on planned industry internships and other partnerships.
+
+## SAMPL7 plans (present through late 2019)
+
+SAMPL7 is planned to include:
+- "SAMPL6 part 2" logP challenge (closed March, 2019)
+- Host-guest binding challenges on Isaacs' TrimerTrip, Gibb's GDCC's, and cyclodextrin derivatives from the Gilson lab
+- Physical property prediction (log D), given pKa values, with data available in summer or Fall 2019; this is a partnership with GSK.
+
+Potentially the logD property prediction could be bumped to early in the SAMPL8 challenge depending on progress of data collection.
+
+### The SAMPL7 physical property challenge
```

Comment:
We don't know what data will be available from GSK collaborators yet, but if there is logD data on both water and another non-polar solvent I would prefer to swich SAMPL7 and SAMPL8 logD challenges, to have non-octanol solvent first.

As discussed in the virtual workshop octanol-water distribution has 3 problems:
1. pKa prediction
2. neutral species partitioning
3. charged species partitioning, as it is known that significant ratios of ions can partition in to the octanol phase.

This 3rd problem would be minimal in a non-polar solvent like dodecane, heptane or cyclohexane, and I believe that should be the next challenge in SAMPL7 aiming for moderate difficulty. We should keep the octanol logD data for SAMPL8.

Reply:
OK, I am adding more discussion along these lines. These are excellent points.